Dockerfile

```
@@ -0,0 +1,31 @@
+FROM oven/bun:1.1.34-alpine
+
+RUN addgroup -S nonroot \
+    && adduser -S nonroot -G nonroot
+
+USER nonroot
+
+WORKDIR /home/nonroot/app
+
+# Copy files to install the dependencies
+COPY package.json bun.lockb .
```

Hadolint

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)